linera-sdk/src/lib.rs

```diff
@@ -79,9 +80,11 @@ pub struct ViewStateStorage<A>(std::marker::PhantomData<A>);
 #[async_trait]
 pub trait Contract: Sized {
     /// Message reports for application execution errors.
-    type Error: Error;
+    type Error: Error + From<serde_json::Error>;
     /// The desired storage backend to use to store the application's state.
     type Storage;
+    /// The initialisation arguments for the contract.
+    type InitArg: DeserializeOwned + Send;
```

Comment:
Thanks for the changes but this one belongs to another PR. We just want to quickly improve the command line of the `linera` tool here